cmd/lassie/internal/wrappedstore.go

```diff
@@ -4,27 +4,30 @@ import (
 	"context"
 	"io"
 
-	carstore "github.com/ipld/go-car/v2/storage"
 	"github.com/ipld/go-ipld-prime/storage"
 )
 
+type ReadableWritableStorage interface {
```

Comment:
this comes up a fair amount -- I wonder if ipld-prime should consider adding it.

Reply:
yes cause I'm considering writing this again elsewhere this week!